stdlib/public/core/Sort.swift.gyb

```diff
@@ -86,7 +90,8 @@ func _sort3<C>(
   C : MutableCollection & RandomAccessCollection
   ${"" if p else ", C.Iterator.Element : Comparable"}
 {
-  // Possible starting states for elements at a, b, and c:
+  // Possible starting orderings for elements at `elements[a]`, `elements[b]`,
+  // and `elements[c]`:
```

Comment:
The change just made the text longer without addressing my point, which I obviously failed to articulate well.  What I don't see is where a, b, and c fall in the sequence below.  For example, if you wrote:

```swift
// 123, 132, 213, 231, 312, 321, 112, 121, 211, 122, 212, 221, 111
// ^                                  ^                   ^
// a                                  b                   c
```

I would understand the scenario.